examples/localvideosnapshot/src/helpers.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var Video = require('twilio-video');
+
+/**
+ * Display local video in the given HTMLVideoElement.
+ * @param {HTMLVideoElement} video
+ * @returns {Promise<void>}
+ */
+function displayLocalVideo(video) {
+  return Video.createLocalVideoTrack().then(function(localTrack) {
+    localTrack.attach(video);
+  });
+}
+
+/**
+ * Take snapshot of the local video from the HTMLVideoElement and render it
+ * in the HTMLCanvasElement.
+ * @param {HTMLVideoElement} video
+ * @param {HTMLCanvasElement} canvas
+ */
+function takeLocalVideoSnapshot(video, canvas) {
+  var context = canvas.getContext('2d');
+  context.clearRect(0, 0, canvas.width, canvas.height);
+  context.drawImage(video, 0, 0, canvas.width, canvas.height);
+}
```

Comment:
I think we need to use `video.width` and `video.height` somewhere to maintain aspect ratio.